README.md

```diff
@@ -13,31 +13,67 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**Adios** is an open-source solution that can generate and/or upload image assets for your Ad Groups. If you already have images Adios will be able to upload them to the Google Ads asset library and link to the corresponding ad groups in bulk. If you don't, then Adios will use AI to generate personalized images based on the specified ad group context (e.g. based on the ad group name).
+
+To generate images Adios uses the recent [Gemini](https://blog.google/technology/ai/google-gemini-ai/) model on Google Cloud's [Vertex AI image generation API](https://cloud.google.com/vertex-ai/docs/generative-ai/image/overview).
+
+## Case study
+
+You can find [Adios case study in German](https://www.thinkwithgoogle.com/intl/de-de/marketing-strategien/automatisierung/tourlane-generative-ki/) (or you can try [Google Translate version](https://www-thinkwithgoogle-com.translate.goog/intl/de-de/marketing-strategien/automatisierung/tourlane-generative-ki/?_x_tr_sl=de&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=in,wapp), just select your preferred language in the top menu).
+
+## The Challenge
+
+Many Google Ads customers have a very granular ads account structure with thousands (sometimes millions) of ad groups (each ad group has a very specific/niche topic). Their marketing teams might have hard times when they want to upload personalized image extensions to all those ad groups.
+More specifically the challenges are to:
+
+- Get high quality relevant image assets for each ad group topic
+- Ads UI does not provide an easy way to upload and manage image assets on this large scale
+
+## The Approach
+
+Our team in a close collaboration with several advertisers developed a custom solution **Adios**.
+
+## The solution
+
+Here are some of the main features of the solution:
 
-To generate images Adios uses Google Cloud's [Vertex AI image generation API](https://cloud.google.com/vertex-ai/docs/generative-ai/image/overview).
+- generate thousands/millions of images with the help of Generative AI on Google Cloud (please note that with small code changes almost any Gen AI API can be used for this purpose)
+- automatically upload and manage image assets on Google Ads. If you already have the assets, this part can be used separately (without generating images).
+- manually validate generated images (this is optional, in case you want to double check the quality of the generated assets before uploading them to your ads)
+- create Google Ads experiment. For each selected campaign Adios can automatically create an A/B test to check if your new assets perform better compared to the previous setup (in terms of click-through rate)
```

Comment:
Is the default experiment setup to compare against the preivous setup or against having no images? I don't fully recall...

Reply:
Previouse one, since the new experiment contains variant A) generated assets on the ad group level, B) same ad group, but without ad group level assets. I'll add this to the README